less/tabzilla.less

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "lib/lib.less";
```

Comment:
I don't think we need to replicate the whole of `lib.less` here. All we should need is `.image-replaced()` and `.at2x()`?